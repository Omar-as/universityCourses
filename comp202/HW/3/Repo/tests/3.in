10
10
**********
*--------*
**---***-*
***--***-*
***--***-*
***--***-*
****-----*
*-********
**--***---
--*******-
5
5 5
8 2
9 3
10 1
10 10
