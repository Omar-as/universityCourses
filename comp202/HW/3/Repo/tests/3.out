42
4
6
3
7
